Source/WebCore/html/MediaElementSession.cpp

```diff
@@ -298,6 +298,10 @@ void MediaElementSession::isVisibleInViewportChanged()
 
     if (m_element.isFullscreen() || m_element.isVisibleInViewport())
         m_elementIsHiddenUntilVisibleInViewport = false;
+
+#if PLATFORM(COCOA) && !HAVE(CGS_FIX_FOR_RADAR_97530095)
```

Comment:
Minor nit: doesn't "have fix for Radar ..." imply `PLATFORM(COCOA)`?

Reply:
It should.

Reply:
But "don't have fix for Radar ..." doesn't. And we want this block to be skipped on non-Cocoa platforms.

Reply:
I meant that Radar is Cocoa-specific so you could use `PLATFORM(COCOA)` as part of the definition of `HAVE_CGS_FIX_FOR_RADAR_97530095` now (in PlatformEnableCocoa.h ?), as you will probably do once we have a fix for rdar://97530095.

But it is a minor nit, so feel free to ignore the suggestion.